res/themes/Dark Blue/client/client.css

```
@@ -0,0 +1,1617 @@
+    background: darkorange;
+}
+
+/* Used for Ranked Buttons only at the moment*/
+QToolButton#rankedPlay
+{
+    color: #E5E7EB;
+    border-radius: 6px;
+    border: 1px solid #4B5563;
+}
+
+QToolButton#rankedPlay::disabled
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: disabled

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)